README.md

```diff
@@ -1 +1,74 @@
-# hello-rails-back-end
\ No newline at end of file
+# Hello Rails-React App
+
+This a Ruby on Rails project is for test the gem with React and display multiple hello words.
+
+## Built With
+
+- ![Ruby](https://img.shields.io/badge/Ruby-20232A?style=for-the-badge&logo=ruby&logoColor=61DAFB)
+
+## Getting Started
+
+### Installation
+
+1. Clone the repo
+
+   ```sh
+   git@github.com:miguelesco/recipe_app.git
+   ```
+
+### Usage
```

Comment:
- [ ] In both Front end and Backend application README, please mention each other repo link. Suppose this is backend then in this README mention frontend application with the repo link and vice-versa.